public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -106,14 +107,24 @@ export function ServicesContent(props: ServicesProps) {
       (must: any) => must?.term?.serviceName == null
     );
     await Promise.all([
-      handleServicesRequest(http, DSL, setTableItems, mode, dataSourceMDSId[0].id),
+      handleServicesRequest(
```

Comment:
this only takes in 7 params. I guess we have an extra undefined before tenant.

Reply:
this takes 8 params
https://github.com/opensearch-project/dashboards-observability/blob/298febd5e33b6afde9d7081227ba51f2d6d13bed/public/components/trace_analytics/requests/services_request_handler.ts#L21-L30

Reply:
yup got confused between this and handleDSLRequest